.twigcs.yaml

```diff
@@ -0,0 +1,49 @@
+# Managed by https://github.com/linkorb/repo-ansible. Manual changes will be overwritten.
+ruleset:
+  syntax:
+    enabled: true
+    severity: error
+
+  tags:
+    enabled: true
+    severity: warning
+
+  empty-block:
+    enabled: true
+    severity: error
+
+  deprecated-function:
+    enabled: true
+    severity: warning
+
+  deprecated-filter:
+    enabled: true
+    severity: warning
+
+  deprecated-test:
+    enabled: true
+    severity: warning
+
+  deprecated-operator:
+    enabled: true
+    severity: warning
+
+  missing-space:
+    enabled: true
+    severity: error
+
+  trailing-space:
+    enabled: true
+    severity: error
+
+  strict-variable:
+    enabled: true
+    severity: error
+
+  ternary:
+    enabled: true
+    severity: error
+
+  not-operator:
+    enabled: true
+    severity: error
```

Comment:
It is part of the files unconditionally generated for QA automation. Might be extended in the future to conditionally generate some based on project type. Not sure if it is worth to make it conditional right now, instead of later on when we've gone over more repositories.